frames/src/main/java/com/checkout/frames/mapper/theme/BillingFormStyleMapper.kt

```diff
@@ -117,6 +117,16 @@ internal class BillingFormStyleMapper : Mapper<PaymentFormTheme, BillingFormStyl
             }
         }
 
+        val countryComponent = from.paymentFormComponents.find {
+            PaymentFormComponentField.Country.name == it.paymentFormComponentField.name
+        }
+        val countryInputStyle = defaultComponentStylesValues[BillingFormFields.Country]
+        countryComponent?.let { component ->
+            provideComponentStyle(component, countryInputStyle, from)?.let { inputComponentStyle ->
+                componentStylesValues[BillingFormFields.Country] = inputComponentStyle
+            }
+        }
+
```

Comment:
Added country components in mapper for theming capabilities to load it because we removed mandatory fields for phone and country